content/en/docs/restaurant/how-to/fast_food/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Use KDS in a fast-food restaurant"
+description: "Refer to the following example procedure for using KDS in a fast-food restaurant."
+lead: ""
+date: 2024-04-19T16:03:34+02:00
+lastmod: 2024-04-19T16:03:34+02:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: ""
+    identifier: "fast_food-9d9626a518b1ff27346cbbc6b86a3513"
+weight: 341
+toc: true
+type: docs
+---
+
+As soon as you've set up the necessary restaurant components in Business Central, you can start taking orders and tracking their status via KDS. Refer to the following example procedure for using KDS in a fast-food restaurant:
+
+#### Prerequisite
+
+- [Set up the KDS]({{< ref "../kds_setup/index.md" >}}) and the Restaurant in Business Central.
+
+#### Procedure
+
+1. Log into the POS
+2. If there are multiple options available, select the fast-food restaurant from the restaurant list.     
+   Typically, fast-food restaurants consist of two stations - one for the kitchen, and the other one for packing. But the available options will depend on your setup in Business Central.
```

Comment:
We can add the following here: For example, you can set up the burger, fryer and dessert preparation areas as separate kitchen stations. In this case, each will receive only their own product preparation requests.